src/AddressRegistry.Projections.Syndication/LinkedFeedProjectionManager.cs

```diff
@@ -0,0 +1,31 @@
+namespace AddressRegistry.Projections.Syndication
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Autofac.Features.OwnedInstances;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Runner;
+
+    public class LinkedFeedProjectionManager<TContext> where TContext : RunnerDbContext<TContext>
```

Comment:
@CumpsD @ridingwolf Got ideas for better name?

Reply:
sounds good tbh...

RelatedFeedsProjectionManager
CombinedFeedsProjectionManager